stock_quant_package_dimension/README.rst

```diff
@@ -7,9 +7,9 @@ Stock Quant Package Dimension
    !! changes will be overwritten.                   !!
```

Comment:
The change in [stock_quant_package_dimension](https://github.com/OCA/stock-logistics-workflow/pull/1165/files#diff-5cc932375429134f87e8cf556c589feeb173fa168716649e0c63036f08305fe4) seems not relevant.